### PR TITLE
Ran linter and fixed all the errors thrown by linter

### PR DESCRIPTION
**Pull request by shrinathprabhu**
# Pull Request Template

Resolves or continues [AR-2328](https://team-1624093970686.atlassian.net/browse/AR-2328).

## Changes

- Ran the linter
- Fixed errors thrown by linter
- Converted all the `script` tags to `script setup` syntax

## Checklist

- [x] The branch name follows the format: `developer/AR-XXX-issue-name`.
- [x] The changes have been tested locally.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -7,7 +7,7 @@
     <title>Developer Dashboard | Arcana Network</title>
 
     <script>
-      window.global = window;
+      window.global = window
     </script>
```
Comment (ajithranka) [inline index.html:11]: Just curious: Why is this needed?

Comment (shrinathprabhu) [reply index.html]: Actually it was added when we were trying to add configuration and polyfills for supporting node builtins in vite. I removed it now. It is not required. Thanks for pointing it out. It didn't came into my notice when removing unused variables and functions

File: src/components/AppHeader.vue [hunk 2]
```diff
@@ -1,3 +1,109 @@
...
+
+import AccountUserIcon from '@/assets/iconography/account-user.svg'
+import ArcanaLogo from '@/assets/iconography/arcana-dark-vertical.svg'
+import ArcanaFavicon from '@/assets/iconography/arcana-favicon.svg'
+import MenuIcon from '@/assets/iconography/menu.svg'
+import CloseIcon from '@/components/icons/CloseIcon.vue'
+import VHeader from '@/components/lib/VHeader/VHeader.vue'
+
+defineProps({
+  hideBanner: Boolean,
```
Comment (ajithranka) [inline src/components/AppHeader.vue:14]: Prefer the object syntax for defining props:

```suggestion
  hideBanner: {
    type: Boolean,
    required: false,
    default: false,
  }
```

File: src/components/ConfigureFooter.vue [hunk 3]
```diff
@@ -1,26 +1,61 @@
+<script lang="ts" setup>
+import VButton from '@/components/lib/VButton/VButton.vue'
+
+defineProps({
+  show: {
+    type: Boolean,
+    default: true,
+  },
+  saveLabel: String,
+  cancelLabel: String,
```
Comment (ajithranka) [inline src/components/ConfigureFooter.vue:10]:
```suggestion
  saveLabel: { type: String, default: 'SAVE' },
  cancelLabel: { type: String, default: 'CANCEL' }
```

Remove the default values from the component:

```diff
- :label="cancelLabel || 'CANCEL'"
+ :label="cancelLabel"
```

File: src/components/icons/CloseIcon.vue [hunk 4]
```diff
@@ -1,17 +1,35 @@
-<template>
-  <svg width="14" height="14" viewBox="0 0 14 14" fill="none" xmlns="http://www.w3.org/2000/svg">
-    <path d="M1 1L13 13" :stroke="color" stroke-width="2" stroke-linecap="round" />
-    <path d="M13 1L1 13" :stroke="color" stroke-width="2" stroke-linecap="round" />
-  </svg>
-</template>
-
 <script lang="ts">
```
Comment (ajithranka) [inline src/components/icons/CloseIcon.vue:1]: Change this to the `script setup` syntax?

**Review by ajithranka — APPROVED** [review]

**Review by karthik-durai — APPROVED** [review]
